src/app/common/services/task-service.coffee

```diff
@@ -671,5 +671,21 @@ angular.module("doubtfire.common.services.tasks", [])
 
     Task.assessExtension(task, taskCommentID, assessment, interceptSuccess, onError)
 
+  taskService.pin = (task, onSuccess, onError) ->
```

Comment:
This needs to move to the Task object - so that it is accessed as task.pin not TaskService.pin(task)

At the moment task objects are setup in the Project service